Makefile.manifest

```diff
@@ -0,0 +1 @@
+build/root/Makefile.manifest
```

Comment:
Why create this symlink?

Reply:
All the Makefiles are now symlinked to `build/root` dir, changed part of https://github.com/kubernetes/kubernetes/pull/45855

Reply:
But why have this symlink here in the root of the repo? Why not always source from `build/root/`?

Reply:
make sense! I was just influenced by other makefiles in the root dir!